lib/ui/semantics.dart

```diff
@@ -7,11 +7,10 @@ part of dart.ui;
 /// The possible actions that can be conveyed from the operating system
 /// accessibility APIs to a semantics node.
 ///
-/// \warning When changes are made to this class, the equivalent APIs in
-///         `lib/ui/semantics/semantics_node.h` and in each of the embedders
-///         *must* be updated.
-/// See also:
-///   - file://./../../lib/ui/semantics/semantics_node.h
+/// > [!Warning]
+/// > When changes are made to this class, the equivalent APIs in
+/// > `lib/ui/semantics/semantics_node.h` and in each of the embedders
```

Comment:
Does this have to be in public doc?

Reply:
Seems like something we can use `//` instead

Reply:
https://github.com/flutter/engine/pull/19817/files turned it from private to public with this non-existing `\warning` thingy. I would also prefer keeping this private, but also don't want to flip-flop between the two every four years or so. Should we once and for all (lol) take this out of the public docs?

Reply:
let's take it out, I missed this part when I reviewed the pr.

Reply:
Done.